src/corehost/cli/hostpolicy.h

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __HOSTPOLICY_H__
+#define __HOSTPOLICY_H__
+
+#include "host_interface.h"
+#include <pal.h>
+
+#if defined(_WIN32)
+    #define HOSTPOLICY_CALLTYPE __cdecl
+#else
+    #define HOSTPOLICY_CALLTYPE
+#endif
+
+struct corehost_initialize_request_t;
+struct corehost_context_contract;
+
+typedef int(HOSTPOLICY_CALLTYPE *corehost_load_fn) (const host_interface_t *init);
+typedef int(HOSTPOLICY_CALLTYPE *corehost_unload_fn) ();
+
+typedef int(HOSTPOLICY_CALLTYPE *corehost_main_fn) (
+    const int argc,
+    const pal::char_t* argv[]);
```

Comment:
C functions with `[]` in the signature are misleading. I prefer to replace the `[]` with `*`.